src/auditor/checks.py

```diff
@@ -367,18 +360,17 @@ def metadata_check(self):
             Where metadata_old is the string from item.metdata, metadata_new
             is path to feature class, and metadata_note is either '',
             'shelved', or 'static'
-        '''
+        """
 
-        metadata_data = {'metadata_fix': 'N',
-                         'metadata_old': '',
-                         'metadata_new': '',
-                         'metadata_note': ''}
+        metadata_data = {'metadata_fix': 'N', 'metadata_old': '', 'metadata_new': '', 'metadata_note': ''}
```

Comment:
**suggestion**: if you like the one property per line formatting (which i do), you can add a trailing comma to the last item and yapf will format it like you had it.

Reply:
I thought I'd caught all the yapf reformats, but I missed this one. Thanks!